apps/www/src/components/ui/organisms/feature-sections/feature-alternate-position-icon/index.tsx

```diff
@@ -28,11 +28,13 @@ export default function FeatureAlternatePositionIcon(props: FeatureAlternatePosi
                 )}
               >
                 <div>
-                  <Icon
-                    {...feature.icon}
-                    className='w-auto mb-4 sm:mb-6 h-14 sm:h-24 lg:h-52 lg:mb-0'
-                    strokeWidth={0.75}
-                  ></Icon>
+                  {feature.icon && (
```

Comment:
Why do you check if feature has icon it should be required and trigger an error if not.